docs/recipes/ComputingDerivedData.md

```diff
@@ -1,10 +1,10 @@
 # Computing Derived Data
 
-[Reselect](https://github.com/faassen/reselect.git) is a simple library for creating memoized, composable **selector** functions. Reselect selectors can be used to efficiently compute derived data from the Redux store. 
+[Reselect](https://github.com/reactjs/reselect.git) is a simple library for creating memoized, composable **selector** functions. Reselect selectors can be used to efficiently compute derived data from the Redux store.
 
 ### Motivation for Memoized Selectors
 
-Let's revisit the [Todos List example](../basics/UsageWithReact.md):
+> The examples in this section are based on the [Redux Todos List example](http://redux.js.org/docs/basics/UsageWithReact.html).
```

Comment:
Oh, I didn't do it on purpose, it's because of a copy paste from the reselect docs. I'll switch it back.